tests/test_utils.py

```diff
@@ -216,3 +219,32 @@ def test_is_valid():
     """Check the function is_valid()."""
     assert not is_valid(None)
     assert is_valid("parameter")
+
+
+def test_create_dir_structure():
+    """Check for directory structure.
+
+    parentdir
+    ├── childdir
+    └── hello.txt
+    """
+    dir_struct = {
+        'name': 'parentdir',
+        'type': 'dir',
+        'contains': [
+            {
+                'name': 'hello.txt',
+                'type': 'file',
+                'contains': "Some dummy text"
+            },
+            {
+                'name': 'childdir',
+                'type': 'dir',
+            }
+        ]}
+    root_path = tempfile.TemporaryDirectory().name
```

Comment:
I'd like to add negative assertion here:

```
assert os.path.exists(root_path)
assert not os.path.exists(os.path.join(root_path, 'parentdir'))
...
...
....
```

Reply:
I think I didn't get that. We are checking for file/dir exists or not.

Reply:
yes, I meant is as a pre test:

1) test that directory/file DOES NOT exist
2) run the cds
3) test that directory/file DOES exist

Reply:
oh okay.